src/creature_instances.c

```diff
@@ -745,43 +745,80 @@ long instf_fart(struct Thing *creatng, long *param)
 
 long instf_first_person_do_imp_task(struct Thing *creatng, long *param)
 {
+    struct CreatureControl* cctrl = creature_control_get_from_thing(creatng);
+    struct PlayerInfo* player = get_my_player();
     TRACE_THING(creatng);
-    MapSlabCoord slb_x = subtile_slab_fast(creatng->mappos.x.stl.num);
-    MapSlabCoord slb_y = subtile_slab_fast(creatng->mappos.y.stl.num);
-    struct SlabMap* slb;
```

Comment:
A while back there was a job done on the entire code base to define as many of the variables/structs on the start of the function, so to be consistent I think this `struct SlabMap* slb;` should be returned here and line 814 turned back to what it was.

Reply:
Just defined, not initialised?

Reply:
yes